### ISSUE TITLE
issue with multiple processes of tablacus

### ISSUE THREAD

**Issue by lazymonkey2** [tablacus/TablacusExplorer#706]
Hello,
I'm using tablacus on win10 x64:

TE64 23.9.13 Microsoft Windows 10 Enterprise 64-bit 22H2 (10.0.19045) IE/11 JS/11.0.16384 it_it 96 AMD Ryzen 5 PRO 3500U w/ Radeon Vega Mobile Gfx Windows Defender

addons:
foldersettings 1.39,multithread 1.29,mouse 1.33,key 1.26,mainmenu 1.16,titlebar 1.09,tabplus 1.96,innerback 1.14,innerforward 1.16,inneraddressbar 1.35,innerfilterbar 1.34,innerup 1.10,innerrefresh 1.07,innernewfolder 1.06,gridlines 1.05,importexplorer 1.11,split 1.26,undoclosetab 1.16,touchex 1.10,findfiles 1.19,delete 1.07,fastlistmode 1.04,favorites 1.06,usercss 1.08,switchhidden 1.05,tabgroups 1.45,sizestatus 1.27,tabposition 1.10

While moving files I notices that there are multiple processes of TE64.exe.
If I close tablacus the processes with worker.js are not terminated.

![image](https://github.com/tablacus/TablacusExplorer/assets/26486758/dc4ba564-c9eb-43ab-b4fc-1c8761303710)

**Comment by ccfs** [comment]
Had you disabled "multi process" add-on BEFORE moving files?

**Comment by lazymonkey2** [comment]
I use "multi process" because I want to keep using tablacus while copying, moving or deleting big folders containing thousands of files, which could take many minutes.

I disabled "multi process" and the issue goes away, but I need this addon:  I don't want to wait for 5 minutes or more, until the operation is completed.

So after disabling "multi process" I tried iinstalling "multi thread", which I thought it was doing the same thing as "multi process", but using threads in place of processes.

Instead it doesn't do what I was expecting: for delete operations it let me keep using tablacus (it doesn't block the ui) but for copy and move operations it does not, and the user interface is blocked until the operation is done (so I'm not sure what's the purpose of copy and move checkboxes in options).

So the issue seems to be in the "multi process" addon, but I cannot disable it.

**Comment by ccfs** [comment]
"multi thread" does not block the ui during copy and move operation, but it does not work for right-button drag-and-drop. I guess that you have got the latter case.

I have just tried enabling "Right button drag-and-drop menu" and the blocking goes away. Please try it to see whether this also works on your computer.

I cannot reproduce the remaining process problem of "multi process" on my computer. So I cannot find out the cause yet.

**Comment by lazymonkey2** [comment]
yes, using "Right button drag-and-drop menu" it doesn't block the UI.
I didn't understand that it was needed for "multi threads" when using right mouse button.
Thanks.
